app/models/chore.rb

```diff
@@ -40,8 +40,12 @@ def due_class(due_date = next_due)
   end
 
   def add_business_days(start_date:, work_days:)
-    raise ArgumentError, 'start_date must be a Date' unless start_date.instance_of?(Date)
-    raise ArgumentError, 'work_days must be a positive integer' unless work_days.is_a?(Integer) && work_days > 0
+    start_date = start_date.to_date
```

Comment:
looks like you have two things going on here. start date and work days. can you separate that logic out more. without looking at any of the code looking at this method you can tell it's starting to get a bit messy